eo-runtime/src/test/java/org/eolang/AtCompositeTest.java

```diff
@@ -95,6 +96,7 @@ private static class Rnd extends PhDefault {
         /**
          * Ctor.
          */
+        @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@c71n93 this is also easily fixable

Reply:
@yegor256 As I understand it is common practice to use `PhDefault::add` in constructor of classes that extends `PhDefault` and created for tests. I don't think that we need to fix it. @maxonfjvipon what do you think?

Reply:
@c71n93 it's not a brightest design so we should fix it anyway. But it's not a part of this task. It should be done in separate PR